src/ralph/settings/base.py

```diff
@@ -124,7 +124,7 @@ def os_env_true(var, default=''):
 TIME_ZONE = 'Europe/Warsaw'
```

Comment:
if you want to use this when installing from Docker or deb you have to pass it as env variable. Could you change it to `TIME_ZONE = os.environ.get('TIME_ZONE', 'Europe/Warsaw')`?